### PR TITLE
Add GCP Atomics

### PR DESCRIPTION
**Pull request by burning-pm**
Created 4 GCP-based atomics

**Details:**
GCP - Delete Activity Event Log
GCP - Create Custom IAM Role
GCP - Delete Bucket
GCP - Delete Service Account Key

**Testing:**
Local testing via PowerShell

**Associated Issues:**

### CODE REVIEW

File: atomics/T1078.004/T1078.004.yaml [hunk 1]
```diff
@@ -49,6 +48,12 @@ atomic_tests:
       if [ -x "$(command -v gcloud)" ]; then exit 0; else exit 1; fi;
     get_prereq_command: |
       echo "Please Install Google Cloud SDK before running this atomic test : https://cloud.google.com/sdk/docs/install"
+  - description: | 
+      Check if user is logged in 
+    prereq_command: |
+      gcloud config get-value account
+    get_prereq_command: |
+      gcloud auth login --no-launch-browser
```
Comment (cyberbuff) [inline atomics/T1078.004/T1078.004.yaml:56]: Moved login as a `dependency` so that authentication is not triggered for every atomic.

Comment (cyberbuff) [reply atomics/T1078.004/T1078.004.yaml:56]: T1098 Execution
T1098-1
T1098-2

Comment (cyberbuff) [reply atomics/T1078.004/T1078.004.yaml:56]: T1485 Execution
T1485-1
T1485-2

File: atomics/T1098/T1098.yaml [hunk 2]
```diff
@@ -945,3 +945,69 @@ atomic_tests:
       else {
           Write-Host -ForegroundColor Yellow "You must store the password of the current user by running the prerequisite commands first"
       }
+- name: GCP - Delete Service Account Key
```
Comment (cyberbuff) [inline atomics/T1098/T1098.yaml:948]: T1098-1
T1098-2

File: atomics/T1485/T1485.yaml [hunk 3]
```diff
@@ -67,3 +67,60 @@ atomic_tests:
     command: |
       cipher.exe /w:C:
     name: command_prompt
+- name: GCP - Delete Bucket
+  description: |
```
Comment (cyberbuff) [inline atomics/T1485/T1485.yaml:71]: T1485-1
T1485-2

**Review by patel-bhavin — APPROVED** [review]

**Comment by patel-bhavin** [comment]
nice work with the PR! Thank you @burning-pm @cyberbuff

**Pull request merged**